src/CommunityVoices/Model/Mapper/Quote.php

```diff
@@ -196,6 +195,72 @@ protected function create(Entity\Media $quote)
         $statement->bindValue(':public_document_link', $quote->getPublicDocumentLink());
         $statement->bindValue(':source_document_link', $quote->getSourceDocumentLink());
 
+        $lastQuoteId = intval($this->conn->lastInsertId());
         $statement->execute();
+
+
+        if ($metaDataArr) {
+            $metaDataId = $this->saveMetaData($metaDataArr);
+            $linkMdTable = "UPDATE `community-voices_quotes`
+                            SET metadata_id = :metadata_id
+                            WHERE media_id = :last_quote_id";
+
+            $statement = $this->conn->prepare($linkMdTable);
+            $statement->bindValue(':metadata_id', $metaDataId);
+            $statement->bindValue(':last_quote_id', $lastQuoteId);
+
+            $statement->execute();
+        }
+    }
+
+    protected function saveMetaData($oberlinMDFields, $metaDataId = null) {
```

Comment:
Could this to the end of the file be in a separate file?  An OberlinMetadata Mapper?  It would be really nice to just say map this array using this mapper.

Reply:
@Sammidysam Yes, but the two mappers would need to interact since the quotes table is using the metadata_id field from the metadata table. Given the two tables being intertwined does it still make sense?

Reply:
Yes, I can see the quote mapper calling the Oberlin Metadata mapper create.  That create on the Oberlin Metadata mapper would return the ID of the created object.  I am figuring this happens otherwhere in the code.  Let me know if you cannot find a case (if you want to find one) and I can try to look.